fizzbuzz.js

```diff
@@ -1,12 +1,21 @@
+const Error = require('./statusCodes.js');
+
 const fizzbuzz = (number) => {
-  if ((number % 3 === 0) && (number % 5 === 0)) {
+  if (typeof number === 'undefined'){
+    return (Error.NullValue);
```

Comment:
so you're checking for undefined, but returning an error saying the user passed a different data type (null). Choose one - either check for null and tell the user they've passed a null value, or check for undefined and tell them they've passed undefined.